.travis.yml

```diff
@@ -66,6 +66,10 @@ before_install:
               sleep 10
               curl localhost:8000
           fi
+    - |
+          wget -qO - http://packages.couchbase.com/ubuntu/couchbase.key | sudo apt-key add -
```

Comment:
You can use HTTPS here.

Reply:
You are right. It was copy/pasted from their installation instructions.